### PR TITLE
changed styling to dynamic styling of team page.

### PR DESCRIPTION
**Pull request by Shivani922**

### CODE REVIEW

File: sass/pages/_team.scss [hunk 1]
```diff
@@ -18,12 +18,27 @@
...
     }
     .team-members {
-      width: calc(25% - 40px);
       float: left;
       text-align: center;
       margin: 0 20px 40px;
+      &.seniors{
+        margin: 0 0 40px;
+      }
+      &.employee{
```
Comment (amitjain24524) [inline sass/pages/_team.scss:30]: in seniorManagerTeam and managerTeam if 6 members add each row 3 + 3 and if 5 first row 3 and the second one is 2 and 2 is centre aligned properly on upper row 3 members.

File: js/jsonFetch.js [hunk 2]
```diff
@@ -203,6 +254,7 @@ function toChunkArray(array, chunkSize){
   return chunkArray;
 }
 let renderOpeningSection = function(openingBlock) {
+  console.log("test");
```
Comment (amitjain24524) [inline js/jsonFetch.js:257]: remove logs

**Review by amitjain24524 — CHANGES_REQUESTED** [review]

File: js/jsonFetch.js (hunk 2 above)
Comment (Shivani922) [reply js/jsonFetch.js:257]: done

File: js/jsonFetch.js [hunk 3]
```diff
@@ -47,47 +48,96 @@ let renderCEOTeam = (data) => {
...
+
+
+let renderSeniorManagerTeam = function(SeniorManagerCount) {
+  if(SeniorManagerCount){
+    let SeniorManagerRows = null;
+    if (window.matchMedia("(max-width: 768px)").matches) {
+      SeniorManagerRows = toChunkArray(SeniorManagerCount, 1);
+    } else if (window.matchMedia("(max-width: 992px)").matches) {
+      SeniorManagerRows = toChunkArray(SeniorManagerCount, 2);
+      alert("maxheight recahed");
```
[the remark this answers is not in the document]
Comment (Shivani922) [reply js/jsonFetch.js:60]: It was just for testing. I removed it.

File: sass/pages/_team.scss (hunk 1 above)
Comment (Shivani922) [reply sass/pages/_team.scss:30]: done

**Pull request merged**